pilot/cmd/pilot-agent/main.go

```diff
@@ -290,6 +292,16 @@ var (
 				defer stsServer.Stop()
 			}
 
+			// Start a local DNS server on 15053, forwarding to DNS-over-TLS server
+			// This will not have any impact on app unless interception is enabled.
+			// We can't start on 53 - istio-agent runs as user istio-proxy.
+			// This is available to apps even if interception is not enabled.
+
+			// TODO: replace hardcoded list with settings.
```

Comment:
".global" is hardcoded in ingress - no point to make it configurable here now.

Fixed the other one - using DNSDomain to avoid yet another redundant setting.